O365.Security.Native.ETW.Debug.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>Microsoft.O365.Security.Native.ETW.Debug</id>
-        <version>4.1.9</version>
+        <version>4.1.10</version>
```

Comment:
thanks for updating the nuget version! please also update the `<releaseNotes>` tab in these three `.nuspec` files to describe your change.